configure.ac

```diff
@@ -199,6 +196,12 @@ if test "$with_c_glib" = "yes"; then
 fi
 AM_CONDITIONAL(WITH_C_GLIB, [test "$have_glib2" = "yes" -a "$have_gobject2" = "yes"])
 
+echo "OpenSSL check"
```

Comment:
But this is the latest I fixed... Maybe I lost the changes when doing the rebase. OpenSSL was not used everyplace when I did the patch. In fact it was not even included for c_glib. C++ had this optionally included. I followed the C++ directions.